homeassistant/components/webhook/__init__.py

```diff
@@ -82,7 +82,9 @@ def async_generate_path(webhook_id: str) -> str:
 
 
 @bind_hass
-async def async_handle_webhook(hass, webhook_id, request):
+async def async_handle_webhook(
+    hass: HomeAssistant, webhook_id: str, request: Request
+) -> Response:
```

Comment:
I added the type hints here to ensure that all webhook methods referenced in this PR are typed.